h2o-core/src/main/java/water/api/PutKeyServlet.java

```diff
@@ -69,9 +73,9 @@ private String paramDestinationKey(HttpServletRequest request, HttpServletRespon
     return keyName != null ? keyName : "func_" + Key.rand();
   }
 
-  private Boolean paramOverwrite(HttpServletRequest request, HttpServletResponse response) {
+  private Boolean paramOverwrite(HttpServletRequest request, HttpServletResponse response, boolean defaultValue) {
```

Comment:
Does it make sense to allow a `null` result if defaultValue is provided? It would probably make sense to use primitive boolean instead